### PR TITLE
 Awesome books application

### PR DESCRIPTION
**Pull request by megha-n-bodke**
- refactor the project to add three new sections on top of the two sections which are already completed: Books list (done), Add book form (done), a new Contact info section, a Navigation bar, and a Footer.
- when you click on any of the links, the main area changes to display only one of the three content sections.
- use only plain JavaScript and CSS to change the contents of the page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,20 +10,67 @@
...
+                <li class="links" id="addBook">Add New</li>
+                <li class="links" id="contact">Contact</li>
+            </ul>
+        </nav>
+    </header>
+    <div class="date-section">
+        <p>
+            13 aug 2022
+        </p>
+    </div>
```
Comment (codepantha) [inline index.html]: - [ ] You have carefully followed the wireframe design as closely as possible but you missed the time at the top-right of the page as shown below
### Wireframe
image

### Your Design
image


- [ ] [OPTIONAL] you are hardcoding the date, but you can make make it dynamic by using this handle library called [luxon.js](https://cdnjs.cloudflare.com/ajax/libs/luxon/3.0.1/luxon.min.js)

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️ 

## Hi team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Correct GitHub Flow
✔️ Descriptive PR
✔️ No linter errors
✔️ Good commit history

## Required Changes ♻️
_Check the comments under the review._

- [ ] In your README file, kindly show users how to set up your project on their local machines. For instance, you may use this 👇 
```
git clone https://github.com/megha-n-bodke/awsome_books.git
```

- [ ] You have a merge conflict. Please try to resolve it 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] I noticed that when a user adds a book, the page reloads upon clicking the' Add' button. If you could avoid that reload, it would greatly improve the UX of your application 😎 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (RolandM99) [reply index.html]: - [ ] As requested by the previous reviewer, make sure that you implement also the time. I'll suggest you use the luxon library. Check [this link](https://moment.github.io/luxon/#/) to see how to use it.

**Review by RolandM99 — CHANGES_REQUESTED** [review]
### Hello Team

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi team 👋, 

Well done implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**